src/Aquifer.Jobs/Subscribers/TranslationMessageSubscriber.cs

```diff
@@ -252,19 +251,38 @@ private async Task TranslateResourceCoreAsync(int resourceContentId,
             throw new InvalidOperationException(
                 $"Aborting translation for Resource Content ID {resourceContentId} because it has no Resource Content Version with Text Media Type in the Draft status.");
 
+        var isAquiferization = Constants.AquiferizationStatuses.Contains(resourceContentVersion.ResourceContent.Status);
+        var awaitingStatus = isAquiferization ? ResourceContentStatus.AquiferizeAwaitingAiDraft : ResourceContentStatus.TranslationAwaitingAiDraft;
+        var completeStatus = isAquiferization ? ResourceContentStatus.AquiferizeAiDraftComplete : ResourceContentStatus.TranslationAiDraftComplete;
+        var editorStatus = isAquiferization ? ResourceContentStatus.AquiferizeEditorReview : ResourceContentStatus.TranslationEditorReview;
+
         if (resourceContentVersion.ResourceContentVersionSnapshots.Count == 0)
         {
             throw new InvalidOperationException(
                 $"Aborting translation for Resource Content ID {resourceContentId} because there are no pre-existing Snapshots for Resource Content Version ID {resourceContentVersion.Id}.");
         }
 
+        var resourceContentLanguage = resourceContentVersion.ResourceContent.Language;
+
+        if (isAquiferization && resourceContentLanguage.Id != _englishLanguageId)
+        {
+            throw new InvalidOperationException(
+                $"Aborting translation for Resource Content ID {resourceContentId} because status indicates Aquiferization but content language is {resourceContentLanguage.EnglishDisplay}.");
```

Comment:
```suggestion
                $"Aborting aquiferization for Resource Content ID {resourceContentId} because status indicates Aquiferization but content language is {resourceContentLanguage.EnglishDisplay}.");
```